CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 0.6.0
```

Comment:
Should add some documentation on how to use and the pitfalls. For example, clusters must not exceed X size where X == (route list size - non-k8s routes)